assets/src/components/conditional-details/test/index.js

```diff
@@ -48,26 +27,20 @@ describe('ConditionalDetails', () => {
 	});
 
 	it('has correct classes', () => {
-		act(() => {
-			render(
-				<ConditionalDetails summary={<div>{'Summary'}</div>}>
-					{'children'}
-				</ConditionalDetails>,
-				container
-			);
-		});
+		let { container } = render(
+			<ConditionalDetails summary={<div>{'Summary'}</div>}>
+				{'children'}
+			</ConditionalDetails>
+		);
 
 		expect(container.querySelector('details')).not.toBeNull();
 		expect(container.querySelector('summary')).not.toBeNull();
 
-		act(() => {
-			render(
-				<ConditionalDetails summary={<div>{'Summary'}</div>}>
-					{[null, null]}
-				</ConditionalDetails>,
-				container
-			);
-		});
+		({ container } = render(
+			<ConditionalDetails summary={<div>{'Summary'}</div>}>
+				{[null, null]}
+			</ConditionalDetails>
+		));
```

Comment:
To allow destructuring without a declaration as per [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#assignment_separate_from_declaration_2:~:text=Note%3A%20The%20parentheses%20(%20...%20)%20around%20the%20assignment%20statement%20are%20required%20when%20using%20object%20literal%20destructuring%20assignment%20without%20a%20declaration.).

<details><summary>Example</summary>

```js
const obj = {
	a: 1,
	b: 2,
};

const obj2 = {
	a: 2,
	b: 3,
};

let { a } = obj;

({ a } = obj2); // It will work without declaring `let a` but ESLint will give an error, so assigning it. 

console.log(a); // 2

```

</details>